components/Map.tsx

```diff
@@ -1,17 +1,36 @@
-import React from "react";
-import { MapContainer, TileLayer, Marker } from "react-leaflet";
+import React, { useEffect } from "react";
+import { MapContainer, TileLayer, Marker, useMap } from "react-leaflet";
 import "leaflet/dist/leaflet.css";
 import styles from "./Map.module.css";
 
-const Map = () => {
+export type Position = {
+  lat: number;
+  lng: number;
+};
+
+const Recenter: React.FC<Position> = ({ lat, lng }) => {
+  const map = useMap();
+  useEffect(() => {
+    map.setView([lat, lng]);
+  }, [lat, lng, map]);
+  return null;
+};
+
+export type MapProps = {
+  position?: Position;
+};
+const Map: React.FC<MapProps> = ({
+  position = { lat: 37.2544, lng: 37.3315 },
```

Comment:
It would be better to get the coordinates of the center location from constants. It is also used in here:

https://github.com/acikkaynak/deprem-yardim-frontend/pull/41/files#diff-715ff5f654014cc344c44ff8e56b1d9a529f3e8ebea0d831fe321dfc2627ae60R4